### PR TITLE
Add duplicity check and fix key validation.

### PR DESCRIPTION
**Pull request by peterivanic**

### CODE REVIEW

**Comment by Filip-Vitkovsky** [comment]
In progress

File: intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java [hunk 1]
```diff
@@ -34,10 +34,20 @@ public void performCheck(@NotNull PsiElement element, @NotNull AnnotationHolder
...
         List<String> keys = new ArrayList<>();
+        List<String> allKeys;
         List<String> leafs = new ArrayList<>();
         if (((YangKeyStmt)element).getKeyArgStr()!= null){
-            key = ((YangKeyStmt)element).getKeyArgStr().getText().replaceAll("\"", "");
-            keys = List.of(key.split(" "));
+            key = ((YangKeyStmt)element).getKeyArgStr().getText().replaceAll("\\s+"," ").replaceAll("\"", "");
+            allKeys = List.of(key.split(" "));
+            for (String k : allKeys){
+                if (!keys.contains(k)){
```
Comment (rovarga) [inline intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java:43]: https://docs.oracle.com/javase/7/docs/api/java/util/Set.html#add(E)

i.e. just use add() and check its return value

File: intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java [hunk 2]
```diff
@@ -34,10 +34,20 @@ public void performCheck(@NotNull PsiElement element, @NotNull AnnotationHolder
...
+                    keys.add(k);
+                }else {
```
Comment (rovarga) [inline intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java:45]: whitespace

File: intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java [hunk 3]
```diff
@@ -34,10 +34,20 @@ public void performCheck(@NotNull PsiElement element, @NotNull AnnotationHolder
         String key = "";
         String config = "";
         List<String> keys = new ArrayList<>();
+        List<String> allKeys;
```
Comment (rovarga) [inline intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java:37]: should be declared inside the if() body only

**Review by rovarga — CHANGES_REQUESTED** [review]

**Pull request closed without merging**